scripts/vim-patch.sh

```diff
@@ -253,9 +253,9 @@ review_pr() {
   echo "✔ Saved pull request diff to '${NEOVIM_SOURCE_DIR}/n${base_name}.diff'."
   echo "${neovim_patch}" > "${NEOVIM_SOURCE_DIR}/n${base_name}.patch"
   echo "✔ Saved full pull request commit details to '${NEOVIM_SOURCE_DIR}/n${base_name}.patch'."
-  hg diff --show-function --git --change "${vim_commit}" > "${NEOVIM_SOURCE_DIR}/${base_name}.diff"
+  git diff "${vim_commit}^\!" > "${NEOVIM_SOURCE_DIR}/${base_name}.diff"
```

Comment:
Per `[gitrevisions(7)](http://git-scm.com/docs/gitrevisions)`, that means "the range with only this commit in it" (the meaning is slightly different than that; see the man page).  It's less cumbersome than, e.g. something like "HEAD..HEAD".